backend/src/main/java/cz/syntaxbro/erpsystem/constants/Permissions.java

```diff
@@ -0,0 +1,7 @@
+package cz.syntaxbro.erpsystem.constants;
+
```

Comment:
constants could be an Enum cuz performance sides.
```suggestion
public enum Permissions {
    READ_REPORTS("READ_REPORTS", 0),
    APPROVE_BUDGETS("APPROVE_BUDGETS", 1),
    VIEW_PROFILE("VIEW_PROFILE", 2);
    
    private final String name;
    private final int value;
    
    Permissions(String name, int value) {
        this.name = name;
        this.value = value;
    }
    
    public String getName() {
        return name;
    }
    
    public int getValue() {
        return value;
    }
}
```

Reply:
I agree to use enum here.